### DISCUSSION TITLE
Any plugin or tools to display photoprism (hosted in raspberry pi ) to Roku or TV ?

### DISCUSSION THREAD

**Discussion by sujiths80** [photoprism/photoprism#3570] [Q&A]
I have photoprism hosted in a Rasperry Pi. I have my harddisk connected to Roku & all my photos & videos in the HDD. I would like to access my photos/videos in photoprism in the TV and looking for an option for the same

**Comment by graciousgrey** [comment]
There is no special plugin yet. As PhotoPrism is a web app any plugin that enables you to open a browser on your TV could work however we didn't have much time to optimize the user interface for use on a TV.
